routes/api.js

```diff
@@ -98,14 +120,66 @@ router.get('/getProgressStatus', (req, res) => {
         });
 });
 
+/**
+ * GET mail policy API
+ * @param  {String} '/mailpcpolicy'                                   URI of the resource
+ * @param  {Function} authenticationHelpers.isAuthOrRedirect          Check if user is authenticated else redirect him back to login
+ * @param  {Function} (req, res)                                      Anonymous function to handle request and response
+ */
 router.get('/mailpcpolicy', authenticationHelpers.isAuthOrRedirect, (req, res) => {
     const email = req.user.email;
     mail.mailOptions.to = email;
     mail.mailOptions.subject = 'Peace Corps Policy';
     mail.mailOptions.html = '<H2> Peace Corps Policy </H2>';
     mail.smtpTransport.sendMail(mail.mailOptions, function(error) {
         error ? res.status(500).json({error: 'Something Went Wrong! Try again later.'}) : res.json({message: 'Mail Sent Succesfully.'});
-    })
+    });
+});
+
+/**
+ * UPDATE progress status API
+ * @param  {String} '/updateProgressStatus'                           URI of the resource
+ * @param  {Function} authenticationHelpers.isAuthOrRedirect          Check if user is authenticated else redirect him back to login
+ * @param  {Function} (req, res)                                      Anonymous function to handle request and response
+ */
+router.put('/updateProgressStatus', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    if(req.body && req.body.stage && req.body.activity) {
+        const currStage = req.body.stage;
+        const currActivity = req.body.activity;
+        localUser.find({
+            where: {
+                email: req.user.email
+            },
+            include: [progress]
+        }, {raw: true})
+            .then(data => {
+                const progressStage = data.progress.stage;
+                const progressActivity = data.progress.activity;
+                if((currStage===progressStage || (currStage-progressStage)===1) && ((currActivity-progressActivity)===1)){
```

Comment:
you are having `currStage-progressStage` & `currActivity-progressActivity` in many places, why not creating a const for it?